readme.md

```diff
@@ -73,6 +79,13 @@ Values: `start` `middle` `end`
 
 Position to truncate the string.
 
+##### preferTruncationOnSpace
+
+Type: `boolean`<br>
+Default: `false`
+
+Truncate the string from a whitespace if it is within 3 characters from the actual breaking point
```

Comment:
What's your reasoning for choosing `3`?

Reply:
nothing specific; just thought that it would be appropriate. Should we ask the user for it?